release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -24,9 +26,20 @@ enum CreateReleaseSubstep {
   increment,
 }
 
-/// Displays all substeps related to the 1st step.
+/// Displays all substeps related to initializing a release.
```

Comment:
Prefer matching the verbage. In this case, the class is called `create` so use that verb over `initializing`